themes/theme-b2b/src/components/Radio.styles.ts

```diff
@@ -2,13 +2,13 @@ import { ThemeComponent, cva } from '@marigold/system';
 
 export const Radio: ThemeComponent<'Radio'> = {
   container: cva(),
-  label: cva('disabled:text-text-disabled'),
```

Comment:
"text-text-disabled" still exists in the span component—intentional?
<img width="1150" alt="Captura de pantalla 2024-01-16 a la(s) 20 18 09" src="https://github.com/marigold-ui/marigold/assets/156671137/dde7e04e-5000-4885-9d9b-fa915c1ba809">

Reply:
Oops, this is actually part of the Select component. I think the question still applies though, right?

Reply:
The above screenshot is actually from the Select component, but the same old class name still exists in this component as well. 
<img width="1156" alt="Captura de pantalla 2024-01-16 a la(s) 20 22 12" src="https://github.com/marigold-ui/marigold/assets/156671137/24af4279-567d-43f8-9ee8-a804ebb50a72">

Reply:
Oh your right this is the old token :) I fixed it, good spotted